editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -199,6 +213,12 @@ pub fn posterize_properties(document_node: &DocumentNode, node_id: NodeId, _cont
 	vec![LayoutGroup::Row { widgets: value }]
 }
 
+pub fn quantize_properties(document_node: &DocumentNode, node_id: NodeId, _context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	let value = number_widget(document_node, node_id, 1, "Levels", NumberInput::new().min(1.).max(1000.).int(), true);
+	let index = number_widget(document_node, node_id, 1, "Fit Fn Index", NumberInput::new().min(0.).max(2.).int(), true);
```

Comment:
Would a dropdown be more suited for this?

Reply:
yes but this is mostly temporary